ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Serialisation/SerialiseNodeToClient.hs

```diff
@@ -164,8 +166,41 @@ instance SerialiseHFC xs
 
 instance SerialiseHFC xs
       => SerialiseNodeToClient (HardForkBlock xs) (GenTxId (HardForkBlock xs)) where
-  encodeNodeToClient = dispatchEncoder `after` (getOneEraGenTxId . getHardForkGenTxId)
-  decodeNodeToClient = fmap (HardForkGenTxId . OneEraGenTxId) .: dispatchDecoder
+  encodeNodeToClient cc v (HardForkGenTxId (OneEraGenTxId txid)) =
```

Comment:
When we talked about this recently, IIRC we concluded that `OneEraGenTxId` should wrap a `ShortByteString` directly instead of an n-ary sum as described in #1028, right?